frontend/src/views/CensusDirectory/components/Project.tsx

```diff
@@ -0,0 +1,80 @@
+import { StaticProject } from "census-projects.json";
+import Link from "next/link";
+import { Project, ProjectResponse } from "src/common/queries/censusDirectory";
+import {
+  ProjectContainer,
+  ProjectDetails,
+  ProjectTitle,
+  ProjectSubmitter,
+  ProjectDesctiption,
+  DetailsContainer,
+  ProjectButtons,
+  StyledButton,
+} from "../styles";
+import DetailItem from "./DetailItem";
+interface Props {
+  project: StaticProject | Project;
+  id?: keyof ProjectResponse;
+}
+
+import EmbeddingButton from "./EmbeddingButton";
+
+import notebookLinks from "census-notebook-links.json";
+
+const Project = ({ project, id }: Props) => {
+  const date = new Date(
+    project.last_updated || project.submission_date || ""
+  ).toLocaleDateString("en-US", {
+    dateStyle: "long",
+  });
+
+  const projectNotebookLinks: [string, string][] | undefined =
+    "notebook_links" in project
+      ? project.notebook_links
+      : notebookLinks[id ?? ""];
+
```

Comment:
possible to move anything above the return to a connect.ts and import via useConnect? If it's dependent on props you can pass those to useConnect